configure.ps1

```diff
@@ -26,7 +26,8 @@ Param (
     [switch]$CleanCache,
```

Comment:
Any chance while you're at it, you could move the strong name key disable code out of build.ps1 and into configure.ps1?  It's something that needs to be configured once per machine, not every build.

Reply:
Yeah I wasn't sure if it needed to be configured unless you want to debug our VSIX?  Or is it safe to disable it on everyone's machine that runs this script?

Reply:
I think it needs to be disabled not only to run the VSIX, but also to run the .NET Framework restore task, the non-ilmerged nuget.exe (maybe even the ilmerged nuget.exe). I'm not sure about running tests. Maybe it's needed for that too.  Basically, I think strong name validation for our 2 keys should be disabled for everyone running the script on Windows.